src/geometry/manifold/algebra/smooth_functions.lean

```diff
@@ -58,56 +56,63 @@ under pointwise multiplication.
 -/
 
 @[to_additive]
-instance smooth_map_semigroup {G : Type*} [semigroup G] [topological_space G]
+instance semigroup {G : Type*} [semigroup G] [topological_space G]
   [charted_space H' G] [has_smooth_mul I' G] :
   semigroup C^∞⟮I, N; I', G⟯ :=
 { mul_assoc := λ a b c, by ext; exact mul_assoc _ _ _,
   ..smooth_map.has_mul}
 
 @[to_additive]
-instance smooth_map_monoid {G : Type*} [monoid G] [topological_space G]
+instance monoid {G : Type*} [monoid G] [topological_space G]
```

Comment:
Strictly these unprimed names aren't needed, lean will generate them for you.

Reply:
I know but lean will put them in the times_cont_mdiff_namespace, whereas they should be in the smooth namespace